### ISSUE TITLE
I am searching NPM archives for an old dependence

### ISSUE THREAD

**Issue by Habibouldmoussa** [npm/documentation#676]
Hello, 
I didn't really find a place to ask my question, so I publish it here 
I have an old medical test management platform, which uses dependence called: Promised information. I have no way of knowing what it is used for, at least I would like to know its usefulness before deleting it and possibly replacing it with another dependence. I would like to know if there is the possibility of finding archives of old dependence unpublished.

NPM registry link: https://registry.npmjs.org/promise-information

What I have already done : 

- Ask Stackoverflow 
- search on GitHub and Gitlab
- search on Google and DuckduckGo in any case all the first pages

**Comment by wraithgar** [comment]
No, once things are unpublished they are deleted and not retained.  I'm also unaware of registry followers out there that retain binaries and keep unpublished packages.
